src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="calendar calendar--start-day--mon calendar--month-length--31">
```

Comment:
class calendar--start-day--mon should be calendar--start-day--sun

Reply:
Checklist p10 said that the calendar must have 31 days and start on Monday